crates/cheatnet/src/runtime_extensions/call_to_blockifier_runtime_extension/panic_data.rs

```diff
@@ -1,12 +1,16 @@
 use cairo_felt::Felt252;
+use conversions::byte_array::ByteArray;
 use regex::Regex;
 
 #[must_use]
 pub fn try_extract_panic_data(err: &str) -> Option<Vec<Felt252>> {
-    let re = Regex::new(r"Got an exception while executing a hint: Hint Error: Execution failed. Failure reason:\s\w*\s\(\'(.*)\'\)\.")
+    let re_felt_array = Regex::new(r"Got an exception while executing a hint: Hint Error: Execution failed\. Failure reason:\s\w*\s\('(.*)'\)\.")
         .expect("Could not create panic_data matching regex");
 
-    if let Some(captures) = re.captures(err) {
+    let re_string = Regex::new(r#"(?s)Got an exception while executing a hint: Hint Error: Execution failed\. Failure reason: "(.*)"\."#)
+        .expect("Could not create panic_data matching regex");
```

Comment:
The messages probably should be different

Reply:
Still i think it's a valid comment, easier to find which regex failed to create.